Package.swift

```diff
@@ -23,13 +24,22 @@ var packageDependencies: [Package.Dependency] = [
 ]
 
 var cGRPCDependencies: [Target.Dependency] = []
+
 #if os(Linux)
 // On Linux, Foundation links with openssl, so we'll need to use that instead of BoringSSL.
 // See https://github.com/apple/swift-nio-ssl/issues/16#issuecomment-392705505 for details.
-packageDependencies.append(.package(url: "https://github.com/apple/swift-nio-ssl-support.git", from: "1.0.0"))
+let isLinux = true
 #else
-cGRPCDependencies.append("BoringSSL")
-#endif
+let isLinux = false
+#endif 
+
+// swift build doesn't pass -Xswiftc flags to dependencies, so here using an environment variable
+// is easiest.
+if ProcessInfo.processInfo.environment.keys.contains("GRPC_USE_OPENSSL") || isLinux {
```

Comment:
Super-nit: could you move `isLinux` before the other check, as it is the more important check, and move the comment from above down here?

```
// On Linux, Foundation links with openssl, so we'll need to use that instead of BoringSSL.
 // See https://github.com/apple/swift-nio-ssl/issues/16#issuecomment-392705505 for details.
```

Reply:
done, lmk if it's still not right.